quint/src/cli.ts

```diff
@@ -123,6 +123,11 @@ const compileCmd = {
         type: 'number',
         default: verbosity.defaultLevel,
       })
+      .option('apalache-version', {
+        desc: 'The version of Apalache to use, if it has to be downloaded',
```

Comment:
iiuc, this would also work if I have multiple versions already downloaded and would like to use one of them, right?
Also, it would be good to add a disclaimer that not all versions of Apalache are compatible to all versions of Quint.
```suggestion
        desc: 'The version of Apalache to use, if no running server is found (using this option may result in incompatibility)',
```